### ISSUE TITLE
Move boundary pores to image edge

### ISSUE THREAD

**Issue by jgostick** [PMEAL/porespy#425]
At present they are 1.5 voxels outside the image

**Comment by jgostick** [comment]
This is more complicated than I thought.  Firstly, it's quite hard to do.  I'm trying to make a function that can 'adjust' the boundary pore locations after the fact, but it's quite complicated.  Secondly, I'm not 100% sure its the right thing to do.  And more practically speaking, I'm not sure it's really worth the trouble.  The boundary pores are typically 3-5 voxels, so this is a tiny drop in the bucket compared to a full image of 500-100 voxels.  Also, if the pores are *not* moved, then the additional distance is still accounted for the conductance values, so it's not like the transport is getting a free ride.  The domain is slightly larger and the DP will be slightly higher...these things will cancel out (assuming that boundary pore conductance is handled correctly).  The only issue is if a user inputs the original image size as the domain length/area, yet the pore network simulation includes the extra length of the boundary regions.  

Bottom line is this: even if I were to include this 'feature', it would be optional, in the form of a function that can be run after the extraction has occurred.  Since it won't be part of the default workflow, I think it's safe to shelve this for now and revisit later (i.e. V2.1)

**Comment by jgostick** [comment]
As per the comments above I think we should close this
